packages/common/src/fastCache.ts

```diff
@@ -0,0 +1,30 @@
+export function getFastCache<V>() {
+  let storage: Record<string, V> = {}
+  let timeouts: Record<string, any> = {}
+  return {
+    get(key: string) {
+      return storage[key]
+    },
+    set(key: string, value: V) {
+      queueMicrotask(() => {
+        if (timeouts[key]) {
+          clearTimeout(timeouts[key])
+        }
+        timeouts[key] = setTimeout(() => {
+          delete storage[key]
+          delete timeouts[key]
+        }, 60 * 60 * 1000)
+        storage[key] = value
+      })
+    },
+    clear() {
+      queueMicrotask(() => {
+        storage = {}
+        for (const key in timeouts) {
+          clearTimeout(timeouts[key])
+        }
+        timeouts = {}
+      })
+    },
+  }
+}
```

Comment:
Should this be the default cache within envelop?

Reply:
This can be a seperate library also.

Reply:
Not sure how to handle TTLs without blocking the main thread

Reply:
yes LRU is probably better

Reply:
Do you think it is better now?